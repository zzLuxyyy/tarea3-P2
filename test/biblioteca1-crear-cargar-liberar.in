crearBiblioteca
crearSocio 12345678 Jim Dear 10 03 2004 
agregarSocioBiblioteca
crearSocio 23456789 Lucia Perez 10 03 2010
agregarSocioBiblioteca
imprimirSociosBiblioteca
crearFecha 01/01/1966
crearLibro 411222335 "Todos los fuegos el fuego" "Julio" "Cortázar" "Ocho cuentos que combinan lo cotidiano con lo fantástico" 1
agregarLibroBiblioteca
crearFecha 01/01/1961
crearLibro 421233446 "La ratesa" "Günter" "Grass" "Una alegoría postnuclear narrada por una rata parlante" 2
agregarLibroBiblioteca
crearFecha 01/01/1968
crearLibro 431244557 "Lo bello y lo triste" "Yasunari" "Kawabata" "Un escritor se reencuentra con una amante del pasado" 1
agregarLibroBiblioteca
imprimirLibrosBiblioteca
liberarBiblioteca
Fin
