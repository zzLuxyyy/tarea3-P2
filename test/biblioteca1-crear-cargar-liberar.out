1>Biblioteca creada exitosamente.
2>Socio de ci 12345678 creado exitosamente.
3>Socio de CI 12345678 agregado a la biblioteca.
4>Socio de ci 23456789 creado exitosamente.
5>Socio de CI 23456789 agregado a la biblioteca.
6>Lista de Socios:
Socio 12345678: Jim Dear
Fecha de alta: 10/3/2004
Géneros favoritos: 
Socio 23456789: Lucia Perez
Fecha de alta: 10/3/2010
Géneros favoritos: 
7>8>9>Libro de ISBN 411222335 agregado a la biblioteca.
10>11>12>Libro de ISBN 421233446 agregado a la biblioteca.
13>14>15>Libro de ISBN 431244557 agregado a la biblioteca.
16>Libro 411222335
Título: Todos los fuegos el fuego
Autor: Julio Cortázar
Descripción: Ocho cuentos que combinan lo cotidiano con lo fantástico
Género: 1
Fecha de edición: 1/1/1966
Libro 421233446
Título: La ratesa
Autor: Günter Grass
Descripción: Una alegoría postnuclear narrada por una rata parlante
Género: 2
Fecha de edición: 1/1/1961
Libro 431244557
Título: Lo bello y lo triste
Autor: Yasunari Kawabata
Descripción: Un escritor se reencuentra con una amante del pasado
Género: 1
Fecha de edición: 1/1/1968
17>Biblioteca liberada exitosamente.
18>Fin.
